README.md

```diff
@@ -13,13 +13,12 @@ Please check out our [documentation](https://dash.applovin.com/documentation/med
 To get started with the demo apps, follow the instructions below:
 
 1. Clone this project and open the DemoApp project in the Unity Editor.
-2. Import the [AppLovin Unity Plugin](https://dash.applovin.com/documentation/mediation/unity/getting-started) files to the project.
-3. Under `File > Build Settings`, click on the mobile platform of choice and click on `Player Settings`.
-4. Update the iOS bundle identifier and/or Android package name with your own unique identifier(s) associated with the application you will create in the MAX dashboard (or already created, if it is an existing app).
-5. Open the `HomeScreen.cs` file under `DemoApp/Assets/Scripts/`.
-6. Update the `MaxSdkKey` value with your AppLovin SDK key associated with your account.
-7. Update the MAX ad unit IDs for the ad formats you would like to test. Each ad format will correspond to a unique MAX ad unit ID you created in the AppLovin dashboard for the bundle id used before.
-8. Under `File > Build Settings`, click on the mobile platform of choice and click `Switch Platform` then `Build` to create a project to test the Demo App.
+2. Under `File > Build Settings`, click on the mobile platform of choice and click on `Player Settings`.
```

Comment:
Maybe we can add a step asking to update to the latest version of the plugin via the integration manager like:

**2. Open the AppLovin Integration manager under `AppLovin > Integration Manager` and update to the latest version of the AppLovin MAX plugin by clicking `Upgrade`**

Reply:
I left on auto-update, so shouldn't they get a popup to update? Or is that saved somewhere outside the project?

Reply:
The setting is saved on the editor preferences, so it is not tied to the project settings. But the auto-update is enabled by default, so new users would always see the prompt.

So, I guess that step might not be needed.